earthfile2llb/converter.go

```diff
@@ -114,7 +114,6 @@ func NewConverter(ctx context.Context, target domain.Target, bc *buildcontext.Da
 		Final:   sts,
 		Visited: opt.Visited,
 	}
-	sts.AddOverridingVarsAsBuildArgInputs(opt.OverridingVars)
```

Comment:
This doesn't look right..?

Reply:
They should be added in via `buildTarget`'s `propagateBuildArgs`, but perhaps there's other places we would need to add them in.

```
if !reserved.IsBuiltIn(bai.Name) {
				c.mts.Final.AddBuildArgInput(bai)
}
```